### DISCUSSION TITLE
mvcpaging-2-1/

### DISCUSSION THREAD

**Discussion by martijnboland** [martijnboland/blog#49] [Blog comments]
# mvcpaging-2-1/

https://blogs.taiga.nl/martijn/posts/mvcpaging-2-1/

**Comment by disqus-comments-migrator** [comment]
<em>Rod commented [on Disqus](https://disqus.com/home/forum/martijnboland/) at december 16 2014, 04:33</em>

---

<p>Hi, mvcpaging support mvc 5 ?</p>
